cmdtest_test.go

```diff
@@ -195,12 +196,33 @@ func TestUpdateToTemp(t *testing.T) {
 		ts := mustReadTestSuite(t, dir)
 		ts.Commands["echo-stdin"] = Program("echo-stdin")
 		ts.Commands["echoStdin"] = InProcessProgram("echoStdin", echoStdin)
-		fname, err := ts.files[0].updateToTemp()
+		f, err := ts.files[0].updateToTemp()
+		defer f.Cleanup()
 		if err != nil {
 			t.Fatal(err)
 		}
-		defer os.Remove(fname)
-		if diff := diffFiles(t, "testdata/good/good.ct", fname); diff != "" {
+		if diff := diffFiles(t, "testdata/good/good.ct", f.Name()); diff != "" {
+			t.Errorf("%s: %s", dir, diff)
+		}
+	}
+}
+
+func TestUpdate(t *testing.T) {
+	for _, dir := range []string{"update"} {
```

Comment:
Why a loop if there's only one value?

Reply:
I did this in order to make adding more tests cases easier. Should I remove the loop?

Reply:
Yes. We can add it back if we need it.

Reply:
All right, done and rebased in 2305b24.